src/main/scala/dpla/ingestion3/mappers/providers/NaraExtractor.scala

```diff
@@ -0,0 +1,256 @@
+package dpla.ingestion3.mappers.providers
+
+import java.net.URI
+
+import dpla.ingestion3.enrichments.{DcmiTypeMapper, VocabEnforcer}
+import dpla.ingestion3.mappers.rdf.DCMIType
+import dpla.ingestion3.mappers.xml.XmlExtractionUtils
+import dpla.ingestion3.model._
+import org.eclipse.rdf4j.model.IRI
+
+import scala.xml.{Node, NodeSeq, XML}
+
+class NaraExtractor(rawData:String) extends Extractor with XmlExtractionUtils with Serializable {
+
+  implicit val xml: NodeSeq = XML.loadString(rawData)
```

Comment:
It means I don't need to pass it explicitly to every method, which reduces noise. We've followed this pattern in the other mappers.